config/development.toml

```diff
@@ -664,7 +664,7 @@ host = "https://localhost:9200"
 [opensearch.auth]
 auth = "basic"
 username = "admin"
-password = "admin"
```

Comment:
can you revert this change?

Reply:
Actually this is a required change for running opensearch locally.